docs/components/scroll-restoration.md

```diff
@@ -81,6 +104,32 @@ Or you may want to only use the pathname for some paths, and use the normal beha
 />
 ```
 
+## `scrollContainer`
+
+By default, this component will capture/restore scroll positions on `window`. If your UI scrolls via a different container element with `overflow:scroll`, you will want to use that instead of `window`. You can specify that via the `scrollContainer` prop using a selector or a `ref`:
+
+```tsx
+// Using a selector
+<ScrollRestoration scrollContainer="#container" />
+```
```

Comment:
Might be nice to make this example line up with the other one by adding

```ts
div id="container">
  {/* scrollable contents */}
</div>
```

Reply:
Good idea!